src/app/components/AddressBox/__tests__/index.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react'
```

Comment:
Can you rename test file to `index.test.tsx` ?

Reply:
Sure. Thanks for catching that. Done.